packages/common-lib/src/index.tsx

```diff
@@ -28,6 +29,7 @@ import * as teacherRegistryService from './services/teacherRegistryService'
 import * as classRegistryService from './services/classRegistryService'
```

Comment:
common lib is wrong place to add Services

Reply:
can you suggest where to add this ?